api/dataset_api.go

```diff
@@ -32,27 +33,30 @@ func (api *DatasetAPI) GetInstances(ctx context.Context, vars url.Values) (insta
 
 // SetImportObservationTaskComplete marks the import observation task state as completed for an instance
 func (api *DatasetAPI) SetImportObservationTaskComplete(ctx context.Context, instanceID string) (isFatal bool, err error) {
-	err = api.Client.PutInstanceImportTasks(ctx, api.ServiceAuthToken, instanceID,
+	_, err = api.Client.PutInstanceImportTasks(ctx, api.ServiceAuthToken, instanceID,
```

Comment:
Sorry I didn't notice this one the first time, also needs reformatting in the same way as the other 2